QV.nlogo

```diff
@@ -319,6 +329,43 @@ to assign-pmp-from-distribution
   )
 end
 
+to assign-correlated-pmp-from-distribution
+  ; beta distribution at low variance is similar to a normal distribution
+  ; assume pmp is normally distributed with mean = 0.5 and stdev = 0.1
+  ; 11/1/2020 adding majority? attribute for voters to determine z score of utility
+  ask voters [
+    let zscore 0
+    (ifelse
+      calibration = "manual" [
+        (ifelse majority? = true
+          [set zscore ((utility - majority-mean-utility) / majority-utility-stdev)]
+          [set zscore ((utility - minority-mean-utility) / minority-utility-stdev)]
+        )
+      ]
+      calibration = "1. prop8-mean>0" [
+        (ifelse majority? = true
+          [set zscore ((utility + 0.5) / 0.28)]
```

Comment:
Never put random numbers like this code with explaining them. Either assign them to an informatively named variable (like on lines 94 and 95) or put a comment on the line explaining what the numbers are and why.